packages/create-vite/template-react/README.md

```diff
@@ -6,3 +6,27 @@ Currently, two official plugins are available:
 
 - [@vitejs/plugin-react](https://github.com/vitejs/vite-plugin-react/blob/main/packages/plugin-react/README.md) uses [Babel](https://babeljs.io/) for Fast Refresh
 - [@vitejs/plugin-react-swc](https://github.com/vitejs/vite-plugin-react-swc) uses [SWC](https://swc.rs/) for Fast Refresh
+
+## Expanding the ESLint configuration
+
+If you are developing a production application, you can install [eslint-plugin-react](https://github.com/jsx-eslint/eslint-plugin-react) and update the config:
```

Comment:
Personally I think the Readme should say something more like: If you plan to develop a production application, consider switching to the TS template, instead of helping people starting apps in JS in 2025...

Reply:
I have a soft spot for JS apps still 😄 But I think it makes sense to link to the TS template here as well. I'll add it along side the eslint-plugin-react setup instructions here.

Reply:
Nevermind, I think it's easier to link to the TS template entirely, and they can read how to setup eslint-plugin-react there.